.changeset/metal-bottles-lay.md

```diff
@@ -0,0 +1,52 @@
+---
+'@backstage/create-app': patch
+---
+
+Replaced EntitySystemDiagramCard with EntityCatalogGraphCard
+
+To make this change to an existing app:
+
+`yarn add @backstage/catalog-graph-plugin`
```

Comment:
Best specify that this should be a dependency of packages/app

Reply:
Done